jbmc/src/janalyzer/janalyzer_parse_options.cpp

```diff
@@ -60,7 +60,7 @@ Author: Daniel Kroening, kroening@kroening.com
 janalyzer_parse_optionst::janalyzer_parse_optionst(int argc, const char **argv)
   : parse_options_baset(JANALYZER_OPTIONS, argc, argv),
     messaget(ui_message_handler),
-    ui_message_handler(cmdline, "JANALYZER " CBMC_VERSION)
+    ui_message_handler(cmdline, std::string("JANALYZER ")+CBMC_VERSION)
```

Comment:
Nit picking: clang-format will ask for space around `+` (applies across this PR)

Reply:
done